rfc3164/syslog_message.go

```diff
@@ -10,7 +10,9 @@ import (
 type syslogMessage struct {
 	prioritySet  bool // We explictly flag the setting of priority since its zero value is a valid priority by RFC 3164
 	timestampSet bool // We explictly flag the setting of timestamp since its zero value is a valid timestamp by RFC 3164
+	sequenceSet  bool
 	priority     uint8
+	sequence     int
```

Comment:
Why `int` and not an unsigned type?

Reply:
No good reason. I've seen this number go higher than a uint8, but it's not clear what the max is. I've changed to uint32 here https://github.com/leodido/go-syslog/pull/22/commits/93a234a1556f935c095d59e258c1f1e49dc0b60b